qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -588,7 +588,8 @@ class AdmissionControllerTest extends BaseSpecification {
         and:
         "Restore sensor"
         orchestrator.scaleDeployment("stackrox", "sensor", 1)
-        orchestrator.waitForPodsReady("stackrox", ["app": "sensor"], 1, 30, 1)
+        orchestrator.waitForSensor()
+        ClusterService.waitForClusterHealthy()
```

Comment:
One consideration (I'm not sure if that applies to this PR). Cluster state in DB sometimes does not represent the actual state. There is some delay. i.e. if you disconnect the sensor, it will take a few mins for the state to change in DB.

I didn't check the test fully, maybe we are already doing that - it would be good to wait that the cluster is unhealthy before scaling the sensor. And then wait for a healthy state.

Reply:
Added check for sensor not healthy after scaling it to 0